package.json

```diff
@@ -7,6 +7,8 @@
     "start": "cross-env NODE_ENV=production gulp build+start --obj-dir-name production-electron",
     "serve": "cross-env NODE_ENV=development gulp build+serve --obj-dir-name development-headless",
     "serve+logging": "cross-env NODE_ENV=development LOGGING=on gulp build+serve --obj-dir-name development-headless",
+    "qbrt": "cross-env NODE_ENV=production gulp build+start --platform qbrt --obj-dir-name production-electron",
+    "qbrt+logging": "cross-env NODE_ENV=production LOGGING=on gulp build+start --platform qbrt --obj-dir-name production-electron",
```

Comment:
These should be NODE_ENV=development, and have another start task that uses qbrt in production. But we're getting to a point where we have a ton of npm commands, and it's starting to get a bit hard to follow.

Maybe rename start to "start:electron" and have "start:qbrt" for production builds, while having "dev:electron" and "dev:qbrt" for development.